internal/buildpackage/buildpackage.go

```diff
@@ -0,0 +1,18 @@
+package buildpackage
+
+import (
+	dist2 "github.com/buildpack/pack/internal/dist"
```

Comment:
Why is this `dist2`?

Reply:
Something automatic by the IDE. Will fix. Good catch!